docker/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:lts-alpine AS build
+WORKDIR /pack
+COPY --chown=1000 . .
+RUN npm pack
+
+FROM node:lts-alpine AS prod
+# hadolint ignore=DL3010
```

Comment:
If you're refering to the hadolint rule `DL3010` which I explicitly ignored here, the reason is because the linter mis-interpreted my intention.

As a more general rule, `COPY` is preferred over `ADD`. However, the rule `DL3010` makes an assumption that if I'm copying a `*.tgz` file into a container image, I want to extract it too. The `ADD` command offers a special feature when adding `*.tgz` files where it can automatically extract its contents for you. The `COPY` does not offer the same feature, it just copies. That is why `hadolint` suggests that I use `ADD` instead of `COPY`. 

However in our specific usage here, we don't want to extract the contents of the *.tgz ourselves; we just want to copy the file into the container. Our intention is to let the `npm install` command we execute next take care of the extraction/installation for us. That is why I've explicitly ignored this rule on this line.